client/web/web.go

```diff
@@ -65,13 +65,43 @@ type Server struct {
 	browserSessions sync.Map
 }
 
+// ServerMode specifies the mode of a running web.Server.
+type ServerMode string
+
+const (
+	// LoginServerMode serves a readonly login client for logging
+	// a node into a tailnet, and viewing a readonly interface of
+	// the node's current Tailscale settings.
+	//
+	// In this mode, API calls are authenticated via platform auth.
+	LoginServerMode ServerMode = "login"
+
+	// ManageServerMode serves a management client for editing
+	// tailscale settings of a node.
+	//
+	// This mode is restricts the app to only being assessible
+	// over Tailscale, and API calls are authenticated via browser
+	// sessions associated with the source's Tailscale identity.
+	ManageServerMode ServerMode = "manage"
```

Comment:
Maybe add that if the user is not authorized to manage this device, they will still get a readonly view?

Reply:
oh yeah good call!

Reply:
done!